src/pages/postgraphile/make-extend-schema-plugin.md

```diff
@@ -535,6 +535,62 @@ const MyRegisterUserMutationPlugin = makeExtendSchemaPlugin(build => {
 Note that the `@pgField` directive here is necessary for PostGraphile to "look
 ahead" and determine what to request from the database.
 
+#### Working with arrays via `json_array_elements`
+
+Here's an example of working with a join table, and bulk inserting multiple records from a graphQL List.
+
+```js
+...
+
+typeDefs: gql`
+  input updatePersonsThingsInput {
+    personId: UUID!,
+    thingIds: [UUID!]!
+  }
+  extend type Mutation {
+    updatePersonsThings(input: updatePersonsThingsInput!): PersonsThingsConnection
+  }
+`,
+resolvers: {
+  Mutation: {
+    updatePersonsThings: async (_query, { input: { personId, thingIds } }, { pgClient }, resolveInfo) => {
+      await pgClient.query("SAVEPOINT graphql_mutation");
+      try {
+        // Ensure proper formatting. This may not be necessary if not modifying the input
+        const elements = JSON.stringify(thingIds.map(thingId => ({ thingId, personId })));
+        
+        // Bulk insert
+        await pgClient.query(`
+          INSERT INTO public.persons_things (person_id, thing_id)
+          SELECT
+            (el->>'personId')::uuid,
+            (el->>'thingId')::uuid
+          FROM json_array_elements($1) el
+        `, [elements]);
+
+        // Build response
+        const { data } = await resolveInfo.graphile.selectGraphQLResultFromTable(
+          sql.fragment`public.persons_things`,
+          (tableAlias, queryBuilder) => {
+            queryBuilder.where(
+              sql.fragment`${tableAlias}.person_id = ${sql.value(personId)}`
+            );
+          }
+        );
+
+        return { data };
```

Comment:
Pass the relevant IDs on to the next layer so `UpdatePersonThingsPayload.personThings` can fetch them:

```suggestion
        // Return data for next layer to use
        return { personThingIds: rows.map((row) => row.id) };
```